src/vocallout.cc

```diff
@@ -85,12 +86,11 @@ int main(int argc, char **argv) {
         r(VOResponse::Error("invalid token"), HTTPResponseCode.Forbidden);
         return;
       }
-      router.stop();
+      router.BreakConnections();
```

Comment:
Nit: I'd still say it's best to do this work in the destructor of `StreamRouter`. If you like this idea, just `auto router = std::make_unique<StreamRouter>(mapping, ws_config)` is the way to go, so you could just do `router = nullptr;` when you don't need it. Don't forget to check that it's not a `nullptr` in other handlers though!

This, of course, if `/stop` is meant to be called only once. If it's part of operational routine to break (some?) connections only to re-create them later, ignore this comment.

Reply:
Will keep '/stop' method for now, maybe we will have some cases to reset all connections (for example if all ASRs are  down and we joined one more), but if it will not became real use-case - will remove `stop()` method and add `router = nullptr`